plug_config.py

```diff
@@ -0,0 +1,21 @@
+import environ
+
+from plugs.manager import PlugManager
+from plugs.plug import Plug
+
+env = environ.Env()
+
+scribe_plug = Plug(
+    name="care_scribe",
+    package_name="git+https://github.com/coronasafe/care_scribe.git",
+    version="@scribe",
+    configs={
+        "TRANSCRIBE_SERVICE_PROVIDER_API_KEY": env(
```

Comment:
If its available from the env, we can fetch it from the env  in the plugin, no need to redefine, just make sure we document this somewhere